packages/vscode-extension/src/devices/IosSimulatorDevice.ts

```diff
@@ -351,15 +355,28 @@ export class IosSimulatorDevice extends DeviceBase {
 
     await this.terminateAnyRunningApplications();
 
-    await exec("xcrun", [
+    if (this.nativeLogsOutputChannel) {
+      this.nativeLogsOutputChannel.dispose();
+    }
+
+    this.nativeLogsOutputChannel = window.createOutputChannel(
+      "Radon IDE (iOS Simulator Logs)",
+      "log"
+    );
+    this.nativeLogsOutputChannel.clear();
+
+    const process = exec("xcrun", [
       "simctl",
       "--set",
       deviceSetLocation,
       "launch",
+      "--console",
```

Comment:
AFAIK using `--console` option would make the launch command hang there for as long as the app process is running. I think we should persist `process` in that case and terminate it in `dispose`. It will likely be terminated by the `shutdown` command we execute there but feel like it'd be safer to do it this way in case we ever change the shutdown procedure.